src/components/QR/index.jsx

```diff
@@ -0,0 +1,53 @@
+/*eslint-disable*/
```

Comment:
Does this disable all linting rules?

Reply:
It disables it for the page. Completely forgot I left that there ooooops.